db/migrate/20190328115622_add_pg_search_dmetaphone_support_functions.rb

```diff
@@ -5,6 +5,7 @@ def self.up
     say_with_time('Adding support functions for pg_search :dmetaphone') do
       execute 'CREATE EXTENSION IF NOT EXISTS fuzzystrmatch'
       execute <<~'SQL'
+        create extension fuzzystrmatch;
```

Comment:
Isn't it exactly what it does in line 6? 
Maybe bring that inside the same `execute` command if needed, but I don't see the point in duplicating it.

Reply:
You're right, I've been staring at this too long today :/